diff -ur ruby-treetop-1.4.4/bin/tt ruby-treetop-1.4.4-nogems/bin/tt
--- ruby-treetop-1.4.4/bin/tt	2010-02-20 01:30:39.000000000 +0100
+++ ruby-treetop-1.4.4-nogems/bin/tt	2010-03-02 16:20:08.889260038 +0100
@@ -1,8 +1,6 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby
 require 'optparse'
-require 'rubygems'
 
-$LOAD_PATH.unshift(File.expand_path(File.dirname(__FILE__) + "/../lib"))
 require 'treetop'
 require 'treetop/version'
 
diff -ur ruby-treetop-1.4.4/lib/treetop/bootstrap_gen_1_metagrammar.rb ruby-treetop-1.4.4-nogems/lib/treetop/bootstrap_gen_1_metagrammar.rb
--- ruby-treetop-1.4.4/lib/treetop/bootstrap_gen_1_metagrammar.rb	2010-02-20 01:30:39.000000000 +0100
+++ ruby-treetop-1.4.4-nogems/lib/treetop/bootstrap_gen_1_metagrammar.rb	2010-03-02 16:22:50.532123974 +0100
@@ -1,16 +1,7 @@
 # This file's job is to load a Treetop::Compiler::Metagrammar and Treetop::Compiler::MetagrammarParser
 # into the environment by compiling the current metagrammar.treetop using a trusted version of Treetop.
 
-require 'rubygems'
-dir = File.dirname(__FILE__)
-
-TREETOP_VERSION_REQUIRED_TO_BOOTSTRAP = '>= 1.1.5'
-
-# Loading trusted version of Treetop to compile the compiler
-gem_spec = Gem.source_index.find_name('treetop', TREETOP_VERSION_REQUIRED_TO_BOOTSTRAP).last
-raise "Install a Treetop Gem version #{TREETOP_VERSION_REQUIRED_TO_BOOTSTRAP} to bootstrap." unless gem_spec
-trusted_treetop_path = gem_spec.full_gem_path
-require File.join(trusted_treetop_path, 'lib', 'treetop')
+require 'treetop'
 
 # Relocating trusted version of Treetop to Trusted::Treetop
 Trusted = Module.new
@@ -20,7 +11,6 @@
 
 # Requiring version of Treetop that is under test
 $exclude_metagrammar = true
-require File.expand_path(File.join(dir, '..', 'treetop'))
 
 # Compile and evaluate freshly generated metagrammar source
 METAGRAMMAR_PATH = File.join(TREETOP_ROOT, 'compiler', 'metagrammar.treetop')
@@ -42,4 +32,4 @@
 #   include Trusted::Treetop::Runtime
 # end
 
-$bootstrapped_gen_1_metagrammar = true
\ No newline at end of file
+$bootstrapped_gen_1_metagrammar = true
diff -ur ruby-treetop-1.4.4/lib/treetop.rb ruby-treetop-1.4.4-nogems/lib/treetop.rb
--- ruby-treetop-1.4.4/lib/treetop.rb	2010-02-20 01:30:39.000000000 +0100
+++ ruby-treetop-1.4.4-nogems/lib/treetop.rb	2010-03-02 16:23:57.301124234 +0100
@@ -1,5 +1,3 @@
-require 'rubygems'
-
 module Treetop
   VALID_GRAMMAR_EXT = ['treetop', 'tt']
   VALID_GRAMMAR_EXT_REGEXP = /\.(#{VALID_GRAMMAR_EXT.join('|')})\Z/o
@@ -8,9 +6,9 @@
 dir = File.dirname(__FILE__)
 
 TREETOP_ROOT = File.join(dir, 'treetop')
-require File.join(TREETOP_ROOT, "ruby_extensions")
-require File.join(TREETOP_ROOT, "runtime")
-require File.join(TREETOP_ROOT, "compiler")
+require "treetop/ruby_extensions"
+require "treetop/runtime"
+require "treetop/compiler"
 
 require 'polyglot'
 Polyglot.register(Treetop::VALID_GRAMMAR_EXT, Treetop)
